crates/binstalk-downloader/Cargo.toml

```diff
@@ -47,6 +47,8 @@ xz2 = "0.1.7"
 # because it uses the same zstd-sys version.
 # Otherwise there will be a link conflict.
 zstd = { version = "0.12.3", default-features = false }
+hyper = "0.14.27"
+once_cell = "1.18.0"
```

Comment:
Please make these dependencies optional and only enabled when `trust-dns` is enabled.